### PR TITLE
Add recommendation File for DC

### PR DESCRIPTION
**Pull request by ENOONE**
A recommendation file for dc is added.

@githubteacher Please review the code and merge the branch

### CODE REVIEW

**Comment by rsjiezhang** [comment]
👍

File: ENOONE.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Washington D.C.
+
+### Place to Visit
+- Place 1
+- Place 2
+- Place 3
+
+### Place to Eat
+- Place 1
```
Comment (rsjiezhang) [inline ENOONE.md]: Place 1's lobster was yummy 🥇

**Review by rsjiezhang — APPROVED** [review]
good to go

File: ENOONE.md (hunk 1 above)
Comment (ENOONE) [reply ENOONE.md]: @rsjiezhang Do you happen to know the address?

Comment (rsjiezhang) [reply ENOONE.md]: @ENOONE sure! :) 750 15th St NW, Washington, DC 20005

**Pull request merged**